MosipNexus/deploy/nexus-ui/delete.sh

```
@@ -0,0 +1,31 @@
+## Does NOT delete the "mosip-nexus" namespace (shared with nexus-server) or
+## touch the nexus-server release.
+
+if [ $# -ge 1 ] && [ -n "$1" ] ; then
+  export KUBECONFIG=$1
+fi
+
+NS=mosip-nexus
+RELEASE=nexus-ui
+
+function deleting_nexus_ui() {
+  read -p "Are you sure you want to delete the $RELEASE helm release in namespace $NS? (Y/n) " yn
```

Shellcheck (0.11.0)

[info] read without -r will mangle backslashes.

(SC2162)